contrib/google.golang.org/grpc/grpc_test.go

```diff
@@ -456,7 +456,6 @@ type rig struct {
 func (r *rig) Close() {
 	r.server.Stop()
 	r.conn.Close()
-	r.listener.Close()
```

Comment:
i realized that listeners are closed in `r.server.Stop()`:  https://github.com/grpc/grpc-go/blob/8c50fc25657c1a4e32b99646afa453fcaccd01b2/server.go#L1413